lib/parser/parser-expr.c

```diff
@@ -99,23 +98,22 @@ log_parser_queue_method(LogPipe *s, LogMessage *msg, const LogPathOptions *path_
 
   success = log_parser_process_message(self, &msg, path_options);
 
+  msg_trace("<<<<<< parser rule evaluation result",
+            evt_tag_str("result", success ? "accepted" : "rejected"),
```

Comment:
[optional]

Same here with simplifying the `result` string.

Reply:
same explanation.